staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -48,6 +48,7 @@ const (
 	kmsTransformerPrefixV1       = "k8s:enc:kms:v1:"
 	kmsPluginConnectionTimeout   = 3 * time.Second
 	kmsPluginHealthzTTL          = 3 * time.Second
```

Comment:
Let's rename this to kmsPluginHealthzNegativeTTL.

Reply:
Done